onnxruntime/core/providers/cpu/nn/conv_transpose.h

```diff
@@ -25,7 +25,14 @@ namespace onnxruntime {
 template <typename T>
 class ConvTranspose : public OpKernel {
  public:
-  ConvTranspose(const OpKernelInfo& info) : OpKernel(info), conv_transpose_attrs_(info) {}
+  ConvTranspose(const OpKernelInfo& info) : OpKernel(info), conv_transpose_attrs_(info) {
+    if (auto_pad == AutoPadType::SAME_UPPER || auto_pad == AutoPadType::SAME_LOWER) {
```

Comment:
You will have to duplicate the auto_pad attribute parsing logic here before checking (unfortunately)

Reply:
I used conv_transpose_attrs_.auto_pad instead. Please review it again. Thank you!

Reply:
LGTM